packages/intrepid/matlab/intrelab/CMakeLists.txt

```diff
@@ -10,21 +10,37 @@
 
 PROJECT(Intrelab)
 CMAKE_MINIMUM_REQUIRED(VERSION 3.10)
+set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
+#set(MATLAB_FIND_DEBUG TRUE)
 
 # --> ACTION: define Trilinos install directory
-#set(TRILINOS_INSTALL_DIR "/Users/pakuber/Compadre/Trilinos-new/pthread/install_release")
+#set(TRILINOS_INSTALL_DIR "/Users/pakuber/Compadre/Trilinos-intrelab/pthread/install_release")
 set(TRILINOS_INSTALL_DIR "/ascldap/users/pakuber/Compadre/Trilinos-intrelab/build/install")
```

Comment:
You should not hard-code something like this.  Instead, set the user can point to the Trilinos install in one of the following ways:

1. Set `-DTrilinos_ROOT=<trilinos-install-prefix>`, or
2. Append `<trilinos-install-prefix>` to the env var `CMAKE_PREFIX_PATH` (separating paths with `:`), or
3. Append `<trilinos-install-prefix>` to the cache var `CMAKE_PREFIX_PATH` (separating paths with `;`)

See where `find_package()` looks in:

* https://cmake.org/cmake/help/v3.22/command/find_package.html#config-mode-search-procedure

No one should have to manually modify a CMakeLists.txt file to configure and build a CMake project.